tools/run_tests/performance/run_worker_php.sh

```diff
@@ -16,13 +16,18 @@
 source ~/.rvm/scripts/rvm
 set -ex
 
-repo=$(dirname $0)/../../..
-
+cd $(dirname $0)/../../..
+repo=$(pwd)
 # First set up all dependences needed for PHP QPS test
 cd $repo
 cd src/php/tests/qps
 composer install
+# Install protobuf C-extension for php
+cd vendor/google/protobuf/php/ext/google/protobuf
```

Comment:
Running php using protobuf C-extension works by setting `-d extension=....../protobuf.so` in the file above. 
Currently I am getting this protobuf.so by build it inside vendor, is this a good way to do it? @stanley-cheung

Reply:
Either way is fine. Building it from vendor or `pecl install protobuf` are both fine. Either way it should be from a "released" tag.

Reply:
4 scenarios run successfully in the performance_adhoc:
[php_protobuf_c_extension_to_cpp_protobuf_sync_unary_ping_pong](https://grpc-testing.appspot.com/view/Performance/job/gRPC_performance_adhoc/482/consoleFull)
[php_protobuf_c_extension_to_cpp_protobuf_sync_streaming_ping_pong](https://grpc-testing.appspot.com/view/Performance/job/gRPC_performance_adhoc/484/consoleFull)
[php_protobuf_php_extension_to_cpp_protobuf_sync_unary_ping_pong](https://grpc-testing.appspot.com/view/Performance/job/gRPC_performance_adhoc/481/consoleFull)
[php_protobuf_php_extension_to_cpp_protobuf_sync_streaming_ping_pong](https://grpc-testing.appspot.com/view/Performance/job/gRPC_performance_adhoc/483/consoleFull)